### PR TITLE
Update the 'HelpUri' for 'Get-ExperimentalFeature'

### PR DESCRIPTION
**Pull request by daxian-dbw**
## PR Summary

Update the 'HelpUri' for 'Get-ExperimentalFeature'.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [ ] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [ ] Not Applicable or can only be tested interactively
    - **OR**
    - [x] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

**Comment by adityapatwardhan** [comment]
The two changes are not related. I think we should not club them together in a single PR.

**Comment by daxian-dbw** [comment]
@adityapatwardhan OK, I will separate them into two PRs.

**Comment by daxian-dbw** [comment]
@adityapatwardhan This PR has been updated to only contains the help uri change.

File: src/System.Management.Automation/engine/ExperimentalFeature/GetExperimentalFeatureCommand.cs [hunk 1]
```diff
@@ -13,7 +13,7 @@ namespace Microsoft.PowerShell.Commands
     /// <summary>
     /// Implements Get-ExperimentalFeature cmdlet.
     /// </summary>
-    [Cmdlet(VerbsCommon.Get, "ExperimentalFeature", HelpUri = "")]
+    [Cmdlet(VerbsCommon.Get, "ExperimentalFeature", HelpUri = "https://go.microsoft.com/fwlink/?linkid=2007351")]
```
Comment (adityapatwardhan) [inline src/System.Management.Automation/engine/ExperimentalFeature/GetExperimentalFeatureCommand.cs]: Please add a test for this by adding an entry at: https://github.com/PowerShell/PowerShell/blob/master/test/powershell/engine/Help/assets/HelpURI/V3Cmdlets.csv

**Review by adityapatwardhan — CHANGES_REQUESTED** [review]

File: src/System.Management.Automation/engine/ExperimentalFeature/GetExperimentalFeatureCommand.cs (hunk 1 above)
Comment (daxian-dbw) [reply src/System.Management.Automation/engine/ExperimentalFeature/GetExperimentalFeatureCommand.cs]: Done. csv file updated.

**Review by adityapatwardhan — APPROVED** [review]

**Review by TravisEz13 — APPROVED** [review]

**Pull request merged**